Sources/Sentry/SentryFileManager.m

```diff
@@ -710,6 +710,11 @@ - (BOOL)createDirectoryIfNotExists:(NSString *)path error:(NSError **)error
     return YES;
 }
 
+- (void)clearDiskState
```

Comment:
`l`: This is a duplicate of the existing `deleteAllFolders`.

Reply:
hah, nice

Reply:
ah not quite though, deleteAllFolders only deletes the sentry subfolder based on the current DSN; my implementation went one level higher and deleted all dsn-based folders. looks like deleteAllFolders is called explicitly from some certain test classes; i'll just remove deleteAllFolders and leave my new one in place since it's getting called from all tests.